### PR TITLE
🐛 Set value of url parameter for fast fetch

### PR DESCRIPTION
**Pull request by aishwaryavora**
Setting value of the 'aanf' url parameter with the actual value of the data attribute of the ad element.

### CODE REVIEW

File: extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js [hunk 1]
```diff
@@ -384,6 +384,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       'rc': this.fromResumeCallback ? 1 : null,
       'rafmt': this.getRafmtParam_(),
       'pfx': pfx ? '1' : '0',
+      'aanf': this.element.getAttribute('data-no-fill'),
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js]: Given a pub could theoretically set this value to anything and pass validation, let's make sure it's a boolean.  Can you change this to:

'aanf': /^(true|false)$/i.test(this.element.getAttribute('data-no-fill')) ? this.element.getAttribute('data-no-fill')  : null,

Please add test that ensures only those values are allowed (not case insensitivity).

Comment (aishwaryavora) [reply extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js]: Done. Thanks!!

File: extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js [hunk 2]
```diff
@@ -525,6 +525,37 @@ describes.realWin(
         });
       });
 
+      it('should contain aanf if have no-fill attribute', () => {
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js]: Let's make the testing a bit more exhaustive.  I suggest a parameterized test pattern.  Something like the following but add more cases.

[{noFill: 'true'}, {noFill: 'ignore', notPresent: true}].forEach(({noFill, notPresent}) => {
   it (notPresent ? `should not contain aanf for ${noFill}` : `should have aanf equal to ${noFill}`, () => {
     // create sticky ad and set no fill value
     return impl.getAdUrl().then((url) => {
        if (notPresent) expect(url).to.not.match(/(\?|&)aanf=/);
        else expect(url).to.match(new RegExp(`\?|&)aanf=${noFill}`(&|$)));
     };
}
});

Comment (aishwaryavora) [reply extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js]: Done. Thanks for the suggestion :)

**Pull request merged**